### ISSUE TITLE
Multiple Profiles in the Same Window

### ISSUE THREAD

**Issue by jlaura** [ubarsc/tuiview#87]
I have 2+ DTMs that I am seeking to compare. To compare, I am attempting to draw a transect and then extract 2+ profiles. I can get a single profile to render and see that the profile tool code has support for different pen colors if the polyline is a list. This leads me to believe that multiple concurrent profiles may be supported.

If so, how does one render two profiles from two layers in the same profile window?

**Comment by gillins** [comment]
When you are displaying 3 bands (in RGB mode) all 3 bands are plotted.

Currently I don't support profiling bands that are not currently displayed because it uses the pixel data it has in memory to create the profile. However, if you feel it should be able to profile other bands I'd be open to a PR to add this...

**Comment by jlaura** [comment]
@gillins Cool! Glad a PR would be welcome. Would you be open to a PR that added the ability to profile simultaneously from different layers?

**Comment by gillins** [comment]
Yes, indeed open to it. I'm interested to see how you plan to achieve this though - if the layer isn't currently being displayed you'll have to read it in at some stage....

**Comment by gillins** [comment]
@jlaura I've just merged https://github.com/ubarsc/tuiview/pull/89. Hopefully this hasn't mucked up any PR your are currently working on...
